src/Core/Style.js

```diff
@@ -137,42 +143,62 @@ function defineStyleProperty(style, category, name, value, defaultValue) {
  * any [valid color
  * string](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value).
  * Default is no value, indicating that no filling needs to be done.
- * @property {Image|Canvas|string} fill.pattern - Defines a pattern to fill the
+ * If the `Layer` is a `GeometryLayer` you could use `THREE.Color`.
+ * @property {Image|Canvas|string|function} fill.pattern - Defines a pattern to fill the
  * surface with. It can be an `Image` to use directly, or an url to fetch the
  * pattern from. See [this
  * example](http://www.itowns-project.org/itowns/examples/#source_file_geojson_raster)
  * for how to use.
- * @property {number} fill.opacity - The opacity of the color or the
+ * @property {number|function} fill.opacity - The opacity of the color or the
  * pattern. Can be between `0.0` and `1.0`. Default is `1.0`.
- *
+ * For a `GeometryLayer`, this opacity property isn't used.
+ * @property {number|function} fill.base_altitude - `GeometryLayer` Style, defines altitude
+ * for each Coordinates.
+ * If `base_altitude` is `undefined`, the original altitude is kept, and if it doesn't exist
+ * then the altitude value is set to 0.
+ * @property {number|function} fill.extrusion_height - `GeometryLayer` Style, if defined,
+ * polygons will be extruded by the specified amount
  * @property {Object} stroke - Lines and polygon edges.
- * @property {string} stroke.color The color of the line. Can be any [valid
+ * @property {string|function} stroke.color The color of the line. Can be any [valid
  * color string](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value).
  * Default is no value, indicating that no stroke needs to be done.
- * @property {number} stroke.opacity - The opacity of the line. Can be between
+ * If the `Layer` is a `GeometryLayer` you could use `THREE.Color`.
+ * @property {number|function} stroke.opacity - The opacity of the line. Can be between
  * `0.0` and `1.0`. Default is `1.0`.
- * @property {number} stroke.width - The width of the line. Default is `1.0`.
+ * For a `GeometryLayer`, this opacity property isn't used.
+ * @property {number|function} stroke.width - The width of the line. Default is `1.0`.
+ * @property {number|function} stroke.base_altitude - `GeometryLayer` Style, defines altitude
+ * for each Coordinates.
+ * If `base_altitude` is `undefined`, the original altitude is kept, and if it doesn't exist
+ * then the altitude value is set to 0.
  *
  * @property {Object} point - Point style.
- * @property {string} point.color - The color of the point. Can be any [valid
+ * @property {string|function} point.color - The color of the point. Can be any [valid
  * color string](https://developer.mozilla.org/en-US/docs/Web/CSS/color_value).
  * Default is no value, indicating that no point will be shown.
- * @property {number} point.radius - The radius of the point, in pixel. Default
+ * For a `GeometryLayer`, this opacity property isn't used.
```

Comment:
This line could be removed too, as it mentions `opacity` property within the `color` property description.

Reply:
removed

Reply:
Thanks